### DISCUSSION TITLE
[Android] PDF Generator

### DISCUSSION THREAD

**Discussion by Eric35470** [beeware/toga#2243] [General]
Hello,
I would like to create **PDF files** in an **Android** application with UI and send them by email.
I started with the **fpdf** library for the development on Windows platform and when I wanted to test it on my Android phone, I get an error because fpdf was not supported.
Do you know if this library or another one is or will be integrated on beeware ?
If not, I will have to use another environment or stop my development.
Thanks in advance,
Eric

**Comment by freakboy3742** [comment]
The list of binary libraries available on Android can be found [here](https://chaquo.com/pypi-7.0/); for a more general description of the state of binary support see the [Android platform notes in the Briefcase documentation](https://briefcase.readthedocs.io/en/stable/reference/platforms/android.html#availability-of-third-party-packages). 

Generally speaking, it is *possible* to support any binary package; however, it depends on someone providing a compilation recipe.

**Reply by Eric35470** [reply]
Thanks a lot
